tests/components/glances/__init__.py

```diff
@@ -208,4 +211,5 @@
             "config": "UU",
         },
     },
+    "uptime": utcnow() - timedelta(days=3, hours=10, minutes=25, seconds=10),
```

Comment:
OK this is bad... It does not make sense to use the same data for mocking and as the expected value.. This needs to be changed so that `MOCK_DATA` gets used for the mock and `HA_SENSOR_DATA` only gets used as expected data. I assume that this was also the intention of the creator.

It looks as these tests have never been setup properly.